support/timepb/README.md

```diff
@@ -0,0 +1,3 @@
+# timepb
```

Comment:
Should this be a README or docs.go? Not sure what is best practice. I think the advantage of docs.go is that it shows up in pkg.go.dev like this for sub-packages:
<img width="1313" alt="image" src="https://user-images.githubusercontent.com/48918/153010197-eef9a5b9-07ba-48dc-aff0-6ffbbcd55f36.png">

Reply:
README shows up in pkg.go.dev as well when you go to the package page. I think we need both. Will add doc.go